src/sqlitestorage.cpp

```diff
@@ -198,63 +198,69 @@ bool SqliteStorage::open()
   sqlite3_busy_timeout( d->mDatabase, 1500 );
 
   /* Create Calendars, Components, etc. tables */
-  query = CREATE_VERSION;
-  sqlite3_exec( d->mDatabase );
+#ifdef Q_OS_UNIX
```

Comment:
Well, we use such an ifdef for other parts of the code related to the ProcessMutex.  I don't think there's any harm in doing it like this, personally.